src/main/resources/messages/BiomeBundle.properties

```diff
@@ -1,30 +1,23 @@
 name=Biome
-biome.formatting.service.name=Biome
 biome.settings.name=Biome
 biome.path.label=Biome CLI Path
 biome.config.path.label=Directory path of biome.json
-biome.formatting.failure=Formatting error
 biome.language.server.not.found=Biome language server is not found, make sure you have @biomejs/biome installed.
-biome.os.not.supported=The Biome CLI package doesn't ship with prebuilt binaries for your platform yet. You can still use the CLI by cloning the biome/tools repo from GitHub, and follow the instructions there to build the CLI for your platform.
 biome.loading=Biome is loading...
 biome.language.server.is.running=Biome server is running
 biome.language.server.is.stopped=Biome server is stopped
 biome.language.server.restarted=Biome server restarted
 biome.interpreter.not.configured=Your node interpreter not configured.
 biome.invalid.pattern=Invalid pattern
-biome.run.format.for.files.label=Run format for &files:
-biome.run.lint.for.files.label=Run lint for &files:
+biome.run.format.for.files.label=Run biome for &files:
 biome.run.format.on.save.label=Run format on &save
 biome.run.safe.fixes.on.save.label=Run safe fixes on &save
 biome.run.unsafe.fixes.on.save.label=Run unsafe fixes on &save
-biome.files.pattern.comment=Use a <a href='https://github.com/isaacs/node-glob#glob-primer'>glob pattern</a>, for example, <code>**/*.{js,ts}</code>
-biome.run.format.on.save.checkbox.on.actions.on.save.page=Run Biome format
-biome.run.safe.fixes.on.save.checkbox.on.actions.on.save.page=Run Biome lint --apply
-biome.run.unsafe.fixes.on.save.checkbox.on.actions.on.save.page=Run Biome  lint --apply-unsafe
-action.ReformatWithBiomeAction.text=Reformat with Biome
-action.ReformatWithBiomeAction.description=Reformat with Biome
-biome.apply.safe.fix.with.biome=Apply safe fixes with Biome
-biome.apply.unsafe.fix.with.biome=Apply unsafe fixes with Biome
-biome.failed.to.format.file=Biome failed to format the file
-biome.failed.to.fix.file=Biome failed to fix the file
+biome.run.format.on.save.checkbox.on.actions.on.save.page=Run Biome check
+biome.run.safe.fixes.on.save.checkbox.on.actions.on.save.page=Run Biome check with --apply
+biome.run.unsafe.fixes.on.save.checkbox.on.actions.on.save.page=Run Biome  check with --apply-unsafe
+biome.run.biome.check.with.features=Running Biome check with {0}
+biome.failted.to.run.biome.check.with.features=Failed to run Biome check with {0} for file {1}
```

Comment:
A tiny typo it should be `biome.failed.to.run.biome.check.with.features`

Reply:
In `Run Biome  check with --apply-unsafe` is an double space right after `Run Biome`.